src/includes/set-release/apple.mdx

```diff
@@ -0,0 +1,15 @@
+The SDK will automatically pick up the release version of your application. If you wish to override it, you can do so with `releaseName`:
+
+```swift
+SentrySDK.start(options: [
+    "dsn": "___PUBLIC_DSN___",
+    "releaseName": "my-project-name@2.3.12"
+])
+```
+
+```objc
+[SentrySDK startWithOptions:@{
+    @"dsn": @"___PUBLIC_DSN___",
+    @"releaseName": @("my-project-name@2.3.12")
```

Comment:
String key is `release`:

https://github.com/getsentry/sentry-cocoa/blob/6fc3fb537c99b8d61854c22928abd96bf843a402/Sources/Sentry/SentryOptions.m#L109-L111

```suggestion
    @"release": @("my-project-name@2.3.12")
```